### PR TITLE
update the neurons for kalliope 0.6

### PR DESCRIPTION
**Pull request by satt105**
update the neurons to kalliope 0.6
If there are any mistakes, let me know.

### CODE REVIEW

**Comment by Sispheor** [comment]
Can you just remove the version flag?
Request is a basic lib used by a lot of library. I'm afraid it won't be able to be installed because another package already installed it previously.

**Comment by satt105** [comment]
I don't think they're gonna need anything like a python librairies. Indeed, I fixed the problem with a debug for the user.

**Comment by Sispheor** [comment]
Yes, request is included with kalliope.
Can you squash your commits?
Thanks

File: Readme.md [hunk 1]
```diff
@@ -85,6 +85,12 @@ Finally, create a event, to stop the music when we unlock the phone
 
 Exit Tasker with the exit menu to be sure all events and task have been saved.
 
+Steps to get your personal Key:
+
+    Go to your personal URL
```
Comment (Sispheor) [inline Readme.md]: Indentation here means "code highlight" in markdown.
Please remove the indentation as it's not code here.

**Review by Sispheor — APPROVED** [review]

**Pull request merged**